proto/storage/node.proto

```diff
@@ -86,11 +84,47 @@ message NodeScan {
 }
 
 message NodeInventory {
-    string                         node_id    = 1 [(gogoproto.moretags) = 'search:"Node ID,store" sql:"pk,type(uuid)"'];
-    string                         node_name  = 2 [(gogoproto.moretags) = 'search:"Node,store"'];
-    google.protobuf.Timestamp      scan_time  = 3 [(gogoproto.moretags) = 'search:"Node Scan Time,store"'];
-    scannerV1.Components           components = 4;
-    repeated scannerV1.Note        notes      = 5;
+    string                         node_name = 1;
+    google.protobuf.Timestamp      scan_time = 2;
+
+    // Components represents a subset of the scannerV1.Components proto message containing only fields required for RHCOS node scanning
+    // We are not using scannerV1.Components here for the following reasons:
+    // - to avoid conflicts between v1 and scannerV1 APIs when generating the code in central/graphql/resolvers/generated.go
+    // - to not expose scanner v1 API over stackrox graphql API
+    Components                     components = 3 [(gogoproto.moretags) = 'sql:"-"'];
+    message Components {
+        string                     namespace = 1;
+        repeated RHELComponent     rhel_components = 2;
+        message RHELComponent {
+            int64               id = 1;
+            string              name = 2;
+            string              namespace = 3;
+            string              version = 4;
+            string              arch = 5;
+            string              module = 6;
+            repeated string     cpes = 7;
```

Comment:
With `rhel_content_sets` we don't need `cpes` anymore.

Reply:
Got it removed in [36de34d](https://github.com/stackrox/stackrox/pull/4392/commits/36de34d14c573894a1020512cdc5aeda73b60d31)